cmd/shortener/api/api.go

```diff
@@ -0,0 +1,79 @@
+package api
```

Comment:
Please look at `Standard Go Project Layout`
https://github.com/golang-standards/project-layout

Reply:
renamed to apiserver